### PR TITLE
Some fixes and examples from Movavi

### PR DESCRIPTION
**Pull request by niktischenko**

### CODE REVIEW

File: src/o0baseauth.cpp [hunk 1]
```diff
@@ -93,16 +93,13 @@ void O0BaseAuth::setUseExternalWebInterceptor(bool useExternalWebInterceptor) {
...
-    }
-    
-    return QByteArray();
+    return replyContent_;
 }
 
 void O0BaseAuth::setReplyContent(const QByteArray &value) {
-    if(replyServer_ != NULL) {
-        return replyServer_->setReplyContent(value);
+    replyContent_ = value;
```
Comment (niktischenko) [inline src/o0baseauth.cpp]: Changes in this commit fixes #114

File: src/o0baseauth.h [hunk 2]
```diff
@@ -117,6 +117,11 @@ public Q_SLOTS:
     /// Set extra tokens found in OAuth response
     void setExtraTokens(QVariantMap extraTokens);
 
+    /// Set local reply server
+    void setReplyServer(O2ReplyServer *server);
+
+    O2ReplyServer * replyServer() const;
+
 protected:
     QString clientId_;
```
Comment (niktischenko) [inline src/o0baseauth.h]: I think protected members are bad idea (#114 proves that). So it will be great to replace them with protected getter / setter

**Comment by niktischenko** [comment]
Anybody?

**Comment by pipacs** [comment]
Thanks! The long delay is entirely my own fault.

**Pull request merged**